### ISSUE TITLE
React: Footer - Choose a language combo box -  tab focus is not going to the Close icon.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#6020]
## Detailed description

Issue:
When user type a letter and try to delete it with close icon, tab focus is not going to the Close icon.

Expected:
Tab focus should go to each and every interactive element and it should be keyboard operable.

Note: Keyboard  focus order issue.

**Comment by ariellalgilmore** [comment]
Assigning to myself as Maruthi Latha is going to work on this fix.

**Comment by lathaMaramganti** [comment]
@Praveen-sr , I had tested this issue on stage , it seems to be working fine. Attaching a video. can you please check ? 

https://user-images.githubusercontent.com/1716987/121546121-b048dd80-ca28-11eb-9b8a-baa7de5a4e4a.mp4

**Comment by Praveen-sr** [comment]
@lathaMaramganti Re tested this on React staging, working as expected. 
cc @RobertaJHahn

**Comment by RobertaJHahn** [comment]
Closing based on Praveen's comment above.
